### PR TITLE
Rebalances Loadout points and new items.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request

	About The Pull Request
Essentially add 'low tier' weapons at the armament tab alongside knives and other types of swords.

Raises the points upwards of 25 for more customization options (And in preparation for an incoming PR that adds implants and organs on the loadout menu) so it should be somewhat better to have spare clothing/items tools but not really having round start power tools and such. Makeshift tools are free but others are quite pricey and it is subject to balance by the Head Coder.

Gives the ability to spawn with a specific type of belt that generally you are able to get on the round start

(TO DO in another PR is making that spawning with a specific item drains money from your total bank account the moment you spawn)

That it for now. Again that is subject to Trilby but if accepted I can move to my next project into adding organ modularity to the loadout menu and implants aswell.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
tweak: Tweaked all of the loadout items

/:cl:

### CODE REVIEW

**Comment by Stutternov** [comment]
I don’t personally believe we should be increasing loadout points if there are:
- weapons
- clothing that supplies benefits that normally cannot be found easily / cost money 
- Handheld computers
- And future planned implants

This is just getting a bit silly. If it was /only/ cosmetics this would not be a critique but I do not feel that it is balanced to increase loadout points in this case.

**Comment by LongSleeves** [comment]
mind leaving the hand radio in? I consider it required gear for those coming for expiditions. I should actually add one to the xenoarch lockers and one to the xenoarcheologist lockers. Once your out of comms range that could mean the difference between you being found or forgotten.

**Comment by Guidesu** [comment]
> mind leaving the hand radio in? I consider it required gear for those coming for expiditions. I should actually add one to the xenoarch lockers and one to the xenoarcheologist lockers. Once your out of comms range that could mean the difference between you being found or forgotten.

But its still in

**Comment by LongSleeves** [comment]
Just saw it. It was moved to tools from utility. Thanks.

**Comment by Guidesu** [comment]
> I don’t personally believe we should be increasing loadout points if there are:
> 
> * weapons
> * clothing that supplies benefits that normally cannot be found easily / cost money
> * Handheld computers
> * And future planned implants
> 
> This is just getting to be a bit much. If it was /only/ cosmetics this would not be a critique but I do not feel that it is balanced to increase loadout points in this case.
> 
> I would sooner suggest loadout points stay 10 and any pieces of clothing that offer no armor, flash protection, or major utility be set to price of 0. That way items that are purely cosmetic have no cost to the player but items that may offer something better than what they normally spawn with, i.e sunglasses, a havelock, etc - all have a loadout cost associated to the item's worth.

Doable

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 1]
```diff
@@ -42,6 +42,30 @@
...
+	cost = 2
+
+/datum/gear/armament/tacboot_knife
+	display_name = "tactical knife"
+	path = /obj/item/tool/knife/tacknife
+	cost = 2
+
+/datum/gear/armament/ritual_knife
+	display_name = "ritual knife"
+	path = /obj/item/tool/knife/boot
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:57]: This is the wrong knife

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 2]
```diff
@@ -42,6 +42,30 @@
...
+	cost = 4
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:58]:
```suggestion
	cost = 3
```

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 3]
```diff
@@ -42,6 +42,30 @@
...
+
+/datum/gear/armament/dagger
+	display_name = "dagger"
+	path = /obj/item/tool/knife/dagger
+	cost = 2
+
+/datum/gear/armament/dagger/ceremonial
+	display_name = "Ceremonial knife"
+	path = /obj/item/tool/knife/dagger/ceremonial
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:67]: This one is captain/streward only do to being a really fancy thing

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 4]
```diff
@@ -42,6 +42,30 @@
...
+
+/datum/gear/armament/ritual_knife
+	display_name = "ritual knife"
+	path = /obj/item/tool/knife/boot
+	cost = 4
+
+/datum/gear/armament/dagger
+	display_name = "dagger"
+	path = /obj/item/tool/knife/dagger
+	cost = 2
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:63]:
```suggestion
	cost = 3
```

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 5]
```diff
@@ -75,3 +99,42 @@
...
+	display_name = "Soteria \"Muramasa\" Katana"
+	path = /obj/item/tool/sword/katana/nano
+	cost = 4
+
+/datum/gear/armament/makeshift/saber
+	display_name = "Decorative Saber"
+	path = /obj/item/tool/sword/saber
+	cost = 3
+
+/datum/gear/armament/makeshift/katana
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:117]: You can make this normally

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 6]
```diff
@@ -75,3 +99,42 @@
 	display_name = "katana \"Saya\" sheath"
 	path = /obj/item/clothing/accessory/holster/saber/saya
 
+/datum/gear/armament/katana
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:102]: No katana's are really powerful

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 7]
```diff
@@ -75,3 +99,42 @@
...
+	display_name = "Katana"
+	path = /obj/item/tool/sword/katana
+	cost = 4
+
+/datum/gear/armament/nano
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:107]: This is the best sword in the game, were not adding it to loadout

File: code/modules/client/preference_setup/loadout/lists/belt.dm [hunk 8]
```diff
@@ -0,0 +1,35 @@
+/datum/gear/belt
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/belt.dm:1]: Church belts should be church locked

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 9]
```diff
@@ -75,3 +99,42 @@
...
+	display_name = "Machete"
+	path = /obj/item/tool/sword/machete
+	cost = 2
+
+/datum/gear/armament/spear
+	display_name = "Steel Spear"
+	path = /obj/item/tool/spear/steel
+	cost = 3
+
+/datum/gear/armament/holster/nt
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:137]: This should be church locked

File: code/modules/client/preference_setup/loadout/lists/belt.dm [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+	display_name = "Absolutism tactical belt"
+	path = /obj/item/storage/belt/security/neotheology
+	cost = 4
+
+/datum/gear/belt/mkbelt
+	display_name = "handmade tool belt"
+	path = /obj/item/storage/belt/utility/handmade
+	cost = 2
+
+/datum/gear/belt/mkbelt
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/belt.dm:28]: this is already present in the medical's loadout section and should be doc locked

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 11]
```diff
@@ -0,0 +1,137 @@
+/datum/gear/tools
+	display_name = "Welding Tool"
+	path = /obj/item/tool/weldingtool
+	sort_category = "Tools"
+	cost = 2
+
+/datum/gear/tools/makeshift_welding
+	display_name = "jury-rigged torch"
+	path = /obj/item/tool/weldingtool/improvised
+	cost = 0
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:10]: No on free tools

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+/datum/gear/tools/screwdriver
+	display_name = "screwdriver"
+	path = /obj/item/tool/screwdriver
+	cost = 2
+
+/datum/gear/tools/screwdriver_makeshift
+	display_name = "screw pusher"
+	path = /obj/item/tool/screwdriver/improvised
+	cost = 0
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:22]: Make the normal screwdriver 1 and remove this verson

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+
+/datum/gear/tools/powercellmedium
+	display_name = "power cell, medium"
+	path = /obj/item/cell/medium/high
+	cost = 3
+
+/datum/gear/tools/ducttape
+	display_name = "duct tape"
+	path = /obj/item/tool/tape_roll
+	cost = 2
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:123]: this should be 1 and remove the web tape

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+	cost = 0
+
+/datum/gear/tools/flashlight
+	display_name = "flashlight"
+	path = /obj/item/device/lighting/toggleable/flashlight
+	cost = 1
+
+/datum/gear/tools/hand_radio
+	display_name = "station bound radio"
+	path = /obj/item/device/radio
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:137]: this should have a cost of 1

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+	cost = 1
+
+//------------
+
+/datum/gear/tools/hammer
+	display_name = "hammer"
+	path = /obj/item/tool/hammer
+	cost = 2
+
+/datum/gear/tools/home_makeshift
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:102]: homewrecker shouldnt be added at all

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+	path = /obj/item/tool/multitool
+	cost = 2
+
+/datum/gear/tools/makeshiftmultitool
+	display_name = "wire frier"
+	path = /obj/item/tool/multitool
+	cost = 0
+
+//------------
+/datum/gear/tools/pick
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:74]: this is the upgraded verson of the basic pickaxe

**Comment by Guidesu** [comment]
image
Round Start skinning knife 
Plus changing values of cosmetic items to zero and leaving armored / utility items as it is or lowering or putting it up.
Anything I did not really saw let me know

File: code/modules/client/preference_setup/loadout/lists/armament.dm [hunk 17]
```diff
@@ -42,6 +42,31 @@
...
+
+/datum/gear/armament/dagger
+	display_name = "dagger"
+	path = /obj/item/tool/knife/dagger
+	cost = 3
+
+/datum/gear/armament/dagger/family
+	display_name = "heirloom dagger"
+	path = /obj/item/tool/knife/heirloom_knife
+	cost = 2
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/armament.dm:68]:
```suggestion
	cost = 1
```

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 18]
```diff
@@ -0,0 +1,114 @@
+/datum/gear/tools
+	display_name = "Welding Tool"
+	path = /obj/item/tool/weldingtool
+	sort_category = "Tools"
+	cost = 2
+
+/datum/gear/tools/makeshift_welding
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:7]: Make shift tools dont belong in loadout

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 19]
```diff
@@ -0,0 +1,114 @@
...
+	path = /obj/item/tool/hammer
+	cost = 1
+
+//------------
+
+
+/datum/gear/tools/powercellsmall
+	display_name = "power cell, small"
+	path = /obj/item/cell/small/high
+	cost = 3
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:94]:
```suggestion
	cost = 2
```

File: code/modules/client/preference_setup/loadout/lists/tools.dm [hunk 20]
```diff
@@ -0,0 +1,114 @@
...
+
+/datum/gear/tools/powercellmedium
+	display_name = "power cell, medium"
+	path = /obj/item/cell/medium/high
+	cost = 3
```
Comment (Trilbyspaceclone) [inline code/modules/client/preference_setup/loadout/lists/tools.dm:99]:
```suggestion
	cost = 2
```

**Pull request merged**